settings.py

```diff
@@ -1562,11 +1562,6 @@
             'instance': 'corehq.apps.change_feed.pillow.get_domain_db_kafka_pillow',
         },
         {
-            'name': 'kafka-ucr-main',
-            'class': 'corehq.apps.userreports.pillow.ConfigurableReportKafkaPillow',
-            'instance': 'corehq.apps.userreports.pillow.get_kafka_ucr_pillow',
-        },
-        {
```

Comment:
yeah. that's what the associated ansible PR would do. would be nice to be able to pass in the partitions here, but that's a little more complicated, especially with how the checkpoints work. 

Also if we do this I (or someone else) would need to go manually set checkpoints for the new pillows if they are changed